### DISCUSSION TITLE
Some latex expressions not working with underscore

### DISCUSSION THREAD

**Discussion by wkjung** [community/community#17373] [Other Feature Feedback, Questions, & Ideas]
Some expressions don't work well with underscores. (Maybe because the markdown renderer kicks in first? related to https://github.com/github/feedback/discussions/16939).

As suggested in https://github.com/github/feedback/discussions/16939, after putting a backslash before each underscore character they work.

Examples:

 `$\mathbb{A}_{B}$ $\mathbb{A}_{B}$` => $\mathbb{A}_{B}$ $\mathbb{A}_{B}$
 `$\mathbb{A}\_{B}$ $\mathbb{A}\_{B}$` => $\mathbb{A}\_{B}$ $\mathbb{A}\_{B}$

`${A}_{B}$ ${A}_{B}$` => ${A}_{B}$ ${A}_{B}$
`${A}\_{B}$ ${A}\_{B}$` => ${A}\_{B}$ ${A}\_{B}$

Another strange thing that I don't get:
`${A}_{B}$` => ${A}_{B}$ 

`${A}_{B}{A}_{B}$` => ${A}_{B}{A}_{B}$

This should be fixed or at least documented on escaping.

**Comment by henrythasler** [comment]
But 

`$\mathbf{A}_H = \mathbf{M}_T \times \mathbf{P}_S$` => $\mathbf{A}_H = \mathbf{M}_T \times \mathbf{P}_S$ 

works while more complex expressions aren't rendered at all:

```
$$\begin{aligned}
\mathbf{A} &= \mathbf{M} \times \mathbf{P}_S\\
\begin{pmatrix}
         A_{Az}\\
         A_{Alt}\\ 
         A_z 
     \end{pmatrix} &=      
    \begin{bmatrix}
         M_{11} & M_{12} & M_{13}\\
         M_{21} & M_{22} & M_{23}\\ 
         M_{31} & M_{32} & M_{33} 
     \end{bmatrix} 
     \times 
    \begin{pmatrix}
         P_{Az}\\
         P_{Alt}\\ 
         1 
     \end{pmatrix}
\end{aligned}$$
```
=>

$$\begin{aligned}
\mathbf{A} &= \mathbf{M} \times \mathbf{P}_S\\
\begin{pmatrix}
         A_{Az}\\
         A_{Alt}\\ 
         A_z 
     \end{pmatrix} &=      
    \begin{bmatrix}
         M_{11} & M_{12} & M_{13}\\
         M_{21} & M_{22} & M_{23}\\ 
         M_{31} & M_{32} & M_{33} 
     \end{bmatrix} 
     \times 
    \begin{pmatrix}
         P_{Az}\\
         P_{Alt}\\ 
         1 
     \end{pmatrix}
\end{aligned}$$

However, escaping the `_` with `\` always fixed the issue for me.
